### PR TITLE
Integrate costmap in velocity filter

### PR DESCRIPTION
**Pull request by ghost**
Adapted base_collision_observer.launch and local_costmap_params.yaml to work with updated cob_collision_velocity_filter

### CODE REVIEW

**Comment by mgruhler** [comment]
@mig-em please crossrefrence :-)

requires ipa320/cob_control#107

File: cob_bringup/tools/base_collision_observer.launch [hunk 1]
```diff
@@ -4,27 +4,18 @@
...
-		<!-- load parameter file -->
-		<rosparam file="$(arg pkg_hardware_config)/$(arg robot)/config/footprint_observer_params.yaml" command="load" />
-	</node>
-
-	<!-- start collision velocity filter node -->
+	<!-- start collision velocity filter node-->
 	<node pkg="cob_collision_velocity_filter" type="collision_velocity_filter" name="collision_velocity_filter" output="screen">
 		<!-- remap incoming variables -->
 		<remap from="~obstacles" to="/local_costmap_node/costmap/obstacles" />
 		<remap from="~teleop_twist" to="/base/collision_velocity_filter/command" />
```
Comment (mgruhler) [inline cob_bringup/tools/base_collision_observer.launch]: fix remappings: remove `obstacles` and change `teleop_twist` accordingly.

**Comment by mgruhler** [comment]
CR: ok

**Comment by mgruhler** [comment]
actually also fixes #231

**Comment by mgruhler** [comment]
Tested on r@w3-3: ok

However, I noticed that this requires a "remap" in the [teleop.yaml](https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_hardware_config/raw3-3/config/teleop.yaml#L29) to `/base/collision_velocity_filter/command` to work with teleop.

Before, we had a `safe` and an `override` topic in the [teleop](https://github.com/ipa320/cob_command_tools/blob/2ba8b26a1b24b061d8db9f4a541a350814d8b773/cob_teleop/ros/src/cob_teleop_v1.cpp#L979-L986) which has been removed by ipa320/cob_command_tools#109. Should we restore this behaviour?

@ipa-nhg @ipa-fmw

**Comment by NadiaHG** [comment]
that means this architecture:
newarch

instead of:
arch

@ipa-mig is it right?

**Comment by mgruhler** [comment]
as discussed in person, this should be a combination, with an extra topic on the teleop which routes through the collision velocity filter

**Comment by destogl** [comment]
@ipa-nhg should this also be done for other platforms?

**Comment by mgruhler** [comment]
@destogl @ipa-nhg This is actually a PR not changing any functionality. Before, the `collision_velocity_filter` subscribed to a `costmap_node`. This seperate node has now been integrated in the `collision_velocity_filter`. During this rework, the costmap layout has been (finally) moved to the new layout. The costmap params have been adapted for your robot (r@w3-5, right?) as well.
But please feel free to test :-)

The architecture change discussed above needs to be performed in the `cob_teleop` node, actually. This needs to be done seperately.

**Comment by destogl** [comment]
@ipa-mig: Yes it is r@w3-5. I will test... I have one big update ahead :)

**Comment by NadiaHG** [comment]
FR: OK

teleop node will be modified to support safe and override topic: https://github.com/ipa320/cob_command_tools/issues/144

**Comment by NadiaHG** [comment]
Test on cob4-5: OK

**Pull request merged**